mapie/multi_label_classification.py

```diff
@@ -0,0 +1,746 @@
+from __future__ import annotations
+from typing import Optional, Union, Tuple, Iterable, cast, Sequence
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.linear_model import LogisticRegression
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.utils import check_random_state
+from sklearn.utils.validation import (
+    indexable,
+    check_is_fitted,
+    _num_samples,
+    _check_y
+)
+
+from ._typing import ArrayLike, NDArray
+from .utils import (
+    check_null_weight,
```

Comment:
You can run `isort --recursive .` to sort all the imports of all the files.

Reply:
done